### DISCUSSION TITLE
External validation with Pmetrics version 2.0.2

### DISCUSSION THREAD

**Discussion by uqenovy** [LAPKB/Pmetrics#156] [Q&A]
Hi

We are looking for running an external validation with Pmetrics version 2.0.2.
Do you have any code to share ? the codes from the PM_tutorial/examples didn't work actually.
Thanks

EN

**Comment by mnneely** [comment]
Which tutorial code did not work for you? 

As far as an external validation, if the original model result is obtained with `run1 <- PM_load(1)`, and new data are in a file in your working directory called "data2.csv", then use this code:

`fit2 <- PM_fit$new(model = run1$model, data = PM_data$new("data2.csv"))`
`fit2$run(prior = 1)`

**Comment by uqenovy** [comment]
Hi Mr Neely
Thank you for your reply.
I used exactly the code that you proposed, without success. I obtained this error message :
--------------------------------------------------------------------------------------------
at line 7706 of file NPprep_120.f (unit = 5, file = 'stdin')
Fortran runtime error : End of file

Error termination. Backtrace:
#0 0x153c6f86e
followed by 8 similar lines

INPUT FILE npag103.inp IS NOT AVAILABLE. THE PROGRAM STOPS. TRY AGAIN AFTER RUNNING THE PREPARATION PROGRAM TO CREATE npag103.inp, AND THEN PUTTING THAT FILE IN THE WORKING DIRECTORY.

HIT ANY KEY TO CONTINUE:
Error in  if (success) : missing value TRUE/FALSE needed
In addition : Warning messages:
1: In file.copy(from = prior(2), to = "inputs")
problem copying ./prior.txt to inputs/prior.txt: No such file or directory
2: In file.remove(prior(2)"
--------------------------------------------------------------------------------

Moreover,  a folder is actually created in the WD, but the run seems not going at the adequate place.

Thank you again for your help.

EN
